komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -311,6 +315,35 @@
               </v-container>
             </v-card>
           </v-tab-item>
+
+          <!--  Tab: Thumbnails  -->
+          <v-tab-item v-if="single">
+            <v-card flat>
+              <v-container fluid>
+                <!-- Upload -->
+                <v-row>
+                  <drop-zone @on-input-change="filesChangeHandler" />
```

Comment:
```suggestion
                  <drop-zone @on-input-change="filesChangeHandler" class="pa-8"/>
```

Sizing of the drop zone is done here